tests/lit.py

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env python
```

Comment:
Is it really needed to add the full lit program in there ? It is already in llvm.